media-tv/xmltv/xmltv-0.6.1.ebuild

```diff
@@ -0,0 +1,301 @@
+		dev-perl/HTTP-Cookies
+		dev-perl/URI
+	)
+	zz-sdjson? (
+		>=dev-lang/perl-5.16
+		dev-perl/DateTime
+		virtual/perl-Digest-SHA
+		dev-perl/HTTP-Message
+		dev-perl/LWP-Protocol-https
+		dev-perl/Try-Tiny
+	)
+	zz-sdjson-sqlite? (
+		>=dev-lang/perl-5.16
+		dev-perl/DateTime
+		dev-perl/DateTime-Format-ISO8601
+		dev-perl/DateTime-Format-SQLite
+		dev-perl/DateTime-TimeZone
+		dev-perl/DBD-SQLite
+		dev-perl/DBI
+		virtual/perl-Digest-SHA
+		dev-perl/File-HomeDir
+		dev-perl/File-Which
+		dev-perl/List-MoreUtils
+		dev-perl/LWP-UserAgent-Determined
+	)
+	tv-check? (
+		dev-perl/Tk
+		dev-perl/Tk-TableMatrix
+	)
+	tv-combiner? ( dev-perl/XML-LibXML )
+	tv-pick-cgi? ( dev-perl/CGI )
+"
+
+PREFIX="/usr"
+
+pkg_setup() {
+	# Uses Data::Manip in various places which can fail
+	# if TZ is still set to Factory as it is in stock gentoo
+	# install media
+	export TZ=UTC
+}
+
+src_prepare() {
+	default
+
+	sed -i \
+		-e "s:\$VERSION = '${PV}':\$VERSION = '${PVR}':" \
+		-e "/^@docs/s:doc/COPYING ::" \
+		Makefile.PL || die
+}
+
+src_configure() {
+	make_config() {
+		# Never except default configuration
+		echo "no"
+
+		# Enable Agentina
+		usex ar
+		# Enable Switzerland Search
+		usex ch-search
+		# Enable Denmark
+		usex dk
+		# Enable Latin America
+		usex dtvla
+		# Enable Europe (DEPRECATED, xmltv.se / dotmedia)
+		usex eu-dotmedia
+		# Enable EU epgdata.com including fanart
+		usex eu-epgdata
+		# Enable Europe (xmltv.se / xmltvse)
+		usex eu-xmltvse
+		# Enable Finland
+		usex fi
+		# Enable Swedish listings in Finland
+		usex fi-sv
+		# Enable France
+		usex fr
+		# Enable Hungary and Romania
+		usex huro
+		# Enable Israel
+		usex il
+		# Enable Iceland
+		usex is
+		# Enable Italy
+		usex it
+		# Enable Italy from DVB-S stream
+		echo "no" # missing Linux::DVB
+		# Enable North America using DataDirect
+		usex na-dd
+		# Enable North America from directv.com
+		usex na-dtv
+		# Enable North America XMLTVListings.com
+		usex na-tvmedia
+		# Enable Portugal EPG from sappo.pt
+		usex pt-meo
+		# Enable Portugal EPG from Vodafone
+		usex pt-vodafone
+		# Enable Sweden
+		usex se-swedb
+		# Enable Sweeden Alternative (Repace with eu_xmltvse)
+		usex se-tvzon
+		# Enable Turkey
+		usex tr
+		# Enable UK fast alternative grabber
+		usex uk-bleb
+		# Enable UK/Ireland TV Guide
+		usex uk-tvguide
+		# Enable Schedules Direct JSON
+		usex zz-sdjson
+		# Enable Schedules Direct JSON (SQLite version)
+		usex zz-sdjson-sqlite
+		# rules to improve episode numbering
+		usex tv-check
+		# Enable combiner
+		usex tv-combiner
+		# Enable CGI support
+		usex tv-pick-cgi
+	}
+
+	pm_echovar=`make_config`
+	perl-module_src_configure
+}
+
+src_install() {
+	# actually make test should be unneeded, but if non na grabbers
+	# start to not install remove comment below
+	#make test
+	#make
+
+	# to bypass build issue
+	#make DESTDIR=${D} install || die "error installing"
+
+	perl-module_src_install
+
+	local i
+	for i in $(grep -rl "${D}" "${D}"); do
```

Comment:
Add a comment above descripting what this does.

Reply:
This looks strange. I think it only installs selected grabers. All grabbers are actually built. I assume this is done to limit program size on embedded systems.